src/core/threads.c

```diff
@@ -262,3 +263,13 @@ void MVM_thread_join_foreground(MVMThreadContext *tc) {
         }
     }
 }
+
+MVMint64 MVM_cpucores() {
```

Comment:
Don't we have a  `MVM_platform_cpu_count`?https://github.com/MoarVM/MoarVM/blob/2eedba88d2b0c08f94dd4426a60d0bc7f8effa13/src/platform/sys.h